src/Android/Xamarin.Android/Samples/Data/StatisticalQuery/StatisticalQuery.cs

```diff
@@ -0,0 +1,188 @@
+            StatisticDefinition statDefinitionVarPop = new StatisticDefinition("POP", StatisticType.Variance, "");
+
+            // Create a definition for count that includes an alias for the output
+            StatisticDefinition statDefinitionCount = new StatisticDefinition("POP", StatisticType.Count, "CityCount");
+
+            // Add the statistics definitions to a list
+            List<StatisticDefinition> statDefinitions = new List<StatisticDefinition>
+                          { statDefinitionAvgPop,
+                            statDefinitionCount,
+                            statDefinitionMinPop,
+                            statDefinitionMaxPop,
+                            statDefinitionSumPop,
+                            statDefinitionStdDevPop,
+                            statDefinitionVarPop
+                          };
+
+            // Create the statistics query parameters, pass in the list of definitions
+            StatisticsQueryParameters statQueryParams = new StatisticsQueryParameters(statDefinitions);
+
+            // If only using features in the current extent, set up the spatial filter for the statistics query parameters
+            if (_onlyInExtentSwitch.Checked)
+            {
+                // Get the current extent (envelope) from the map view
+                Envelope currentExtent = _myMapView.GetCurrentViewpoint(ViewpointType.BoundingGeometry).TargetGeometry as Envelope;
+
+                // Set the statistics query parameters geometry with the envelope
+                statQueryParams.Geometry = currentExtent;
+
+                // Set the spatial relationship to Intersects (which is the default)
+                statQueryParams.SpatialRelationship = SpatialRelationship.Intersects;
+            }
+
+            // If only evaluating the largest cities (over 5 million in population), set up an attribute filter
+            if (_onlyBigCitiesSwitch.Checked)
+            {
+                // Set a where clause to get the largest cities (could also use "POP_CLASS = '5,000,000 and greater'")
+                statQueryParams.WhereClause = "POP_RANK = 1";
+            }
+
+            // Execute the statistical query with these parameters and await the results
+            StatisticsQueryResult statQueryResult = await _worldCitiesTable.QueryStatisticsAsync(statQueryParams);
+
+            // Display results in a list in a dialog
+            var statsList = statQueryResult.FirstOrDefault().Statistics.ToList();
+            ShowStatsList(statsList);
+        }
+
+        private void ShowStatsList(IList<KeyValuePair<string, object>> stats)
+        {
+            // Create a list of statistics results (field names and values) to show in the list
+            IList<string> statInfoList = new List<string>();
+            foreach (var kvp in stats)
+            {
+                // If the value is null, display "--"
+                var value = "--";
```

Comment:
I don't know if it needs to change, but I think having a variable named value taking its value from another property named 'value' might result in heightened cognitive load. This might just be a 'just me' thing, but I've read technical publications with lots of similarly-named objects and had trouble grasping the intent as a result.